spec/send_sms_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'send_sms'
+
+describe SendSms do
+
+  let(:phonenumber) { double :phonenumber }
+
+  it "repsonds to send_confirmation_sms" do
+    expect(subject).to respond_to(:send_confirmation_sms)
+  end
+
+  # Unsure if necessary (or possible) given Twilio's own tests/error raising.
+  # it "send_confirmation_sms creates sms using client" do
+  #   expect(subject.send_confirmation_sms(phonenumber)).to be_a Twilio::REST::Message
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)